keyboards/model_f_labs/xwhatsit_util_comm.h

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2020 Purdea Andrei
+ * Copyright 2021 Matthew J Wolf
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#ifndef RAW_EPSIZE
```

Comment:
this looks like it only sets it locally, which means this isn't picked up by the core.  Eg, it's not actually setting the raw ep size

Reply:
RAW_EPSIZE is being used multiple items in keyboards/model_f_labs/xwhatsit_util_comm.c . The code is need to enable the use of a utility for the keyboards, [pandrew-xwhatsit-utility](https://github.com/matthew-wolf-n4mtt/pandrew-xwhatsit-utility)

Reply:
It might be better to simply include "usb_descriptor.h" as that will pull in the raw_epsize define, and keep it consistent if it changes.